City-of-Helsinki/drupal-hdbt
# UHF-4455 & UHF-3454 High school search map embed

# High school search map embed [UHF-4455](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4455) & [UHF-3454](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3454)
In addition to list view of high school search results, we added a map view of the same results.

## What was done
* A new iframe including the results on map was added.
* The list and map results were separated to two tabs and a javascript code was added to make it possible for the user to alternate between the two views.

## How to test
* Check instructions from the https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/94 PR since map view for unit searches is not implemented to other variants than high school search yet.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/94
